docs/guides/export/export-netdata-metrics-graphite.md

```diff
@@ -0,0 +1,184 @@
+<!--
+title: Export and visualize Netdata metrics in Graphite 
+description: Use Netdata to collect and export thouands of metrics to Graphite for long-term storage or further analysis.
```

Comment:
```suggestion
description: Use Netdata to collect and export thousands of metrics to Graphite for long-term storage or further analysis.
```

Reply:
Done! Good catch—the one area I forgot to copy-edit.